librz/include/rz_util/rz_subprocess.h

```diff
@@ -72,8 +72,8 @@ RZ_API void rz_subprocess_free(RzSubprocess *proc);
 RZ_API RzSubprocessWaitReason rz_subprocess_wait(RzSubprocess *proc, ut64 timeout_ms);
 RZ_API void rz_subprocess_kill(RzSubprocess *proc);
 RZ_API int rz_subprocess_ret(RzSubprocess *proc);
-RZ_API char *rz_subprocess_out(RzSubprocess *proc);
-RZ_API char *rz_subprocess_err(RzSubprocess *proc);
+RZ_API char *rz_subprocess_out(RzSubprocess *proc, int *length);
```

Comment:
what about
```suggestion
RZ_API char *rz_subprocess_out(RzSubprocess *proc, size_t *length);
```
?

Reply:
i thought about this, but RzStrbuf does not use size_t.

Reply:
:/ ok